src/components/atoms/Heading/Heading.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC, HTMLAttributes } from 'react';
+import styled from 'styled-components';
+import Text from '../Text/Text';
+import { typography } from '../../../constants/typography';
+
+const {
+  sizes: { heading: headingSizes },
+} = typography;
+
+interface IStyledHeadingProps extends HTMLAttributes<HTMLHeadingElement> {
+  as: 'h1' | 'h2' | 'h3' | 'h4';
+}
+
+const StyledHeading = styled(Text)<IStyledHeadingProps>`
+  font-size: ${({ as = 'h3' }) => headingSizes[as]};
+  letter-spacing: -0.5px;
+`;
```

Comment:
no line-height set here? and no bottom margin like in `<Text />`?

Reply:
The headings have a default `margin` applied.

I believe it comes from the browser as [`nomarlize.css`](https://necolas.github.io/normalize.css/8.0.1/normalize.css) doesn't override the default styles for headings 🤔

Happy to set it to `margin: 0` and have a `mb` prop instead 👍🏻

Reply:
I think that would be more predictable than relying on browser defaults. What line heights are we using?

Reply:
My idea was to set a [global `line-height`](https://github.com/zopaUK/react-components/blob/9ed9c7ba496f5a053fd59942e2d7356d38958d2d/src/constants/typography.ts#L24), however, I noticed I forgot 🤦🏻‍♂️

For now, I just set a relative value, [`1.6`](https://github.com/zopaUK/react-components/blob/9ed9c7ba496f5a053fd59942e2d7356d38958d2d/src/constants/typography.ts#L24),  happy to know if there's a better approach though 👍🏻

For instance, for buttons, I didn't change the `font-size` and `line-height` as it seemed tricky to align them right 😓

Reply:
That would be a possible solution. However, one scenario where I see it getting problematic is if a project has a global line-height already set. If you have it in the component, it's usually enough to overwrite global line-height which allows you to gradually introduce the components without having to mess with global styles